validation-test/Evolution/test_backward_deploy_struct.swift

```diff
@@ -9,25 +9,19 @@ var BackwardDeployStructTest = TestSuite("BackwardDeployStruct")
 
 BackwardDeployStructTest.test("ResilientStruct") {
   if getVersion() == 1 {
-    // FIXME: IRGen inserts the metadata load outside the version check
```

Comment:
There was a second spot where I had this workaround in this directory, can you grep for it (sorry on my phone right now)